sample/SampleHost/Functions.cs

```diff
@@ -33,5 +36,18 @@ public static class Functions
         {
             Console.WriteLine($"Processed work item {workItem.ID}");
         }
+
+        public static async Task ProcessWorkItem_ServiceBus(
+            [ServiceBusTrigger("test-items", AccessRights.Listen)] WorkItem item,
+            string messageId,
+            int deliveryCount,
+            ILogger log)
+        {
+            log.LogInformation($"Processing ServiceBus message (Id={messageId}, DeliveryCount={deliveryCount})");
```

Comment:
I used this sample function to repro the issue. Before my changes, if I introduced a 7 minute delay in this function, the lock would be lost and I'd get retries without any error messages. With my changes the MessageLockLostException from SB is logged to console.